src-java/kilda-model/src/main/java/org/openkilda/model/Flow.java

```diff
@@ -274,6 +274,14 @@ public final void addPaths(FlowPath... paths) {
                 .findAny();
     }
 
+    /**
+     * Checks whether path is is orphaned or not.
+     */
+    public final boolean isValidPath(PathId pathId) {
```

Comment:
Em... it is a more limited copy of `isActualPathId()`. Why we need it?

Reply:
good catch